tests/header.rs

```diff
@@ -4,7 +4,6 @@ mod tests {
     #[test]
     fn read_header_p1() {
         let dstv = Dstv::from_file("./tests/data/P1.nc");
-        assert_eq!(dstv.is_ok(), true);
```

Comment:
I did that, because the unwrap will cause the test to fail if the file did not read (and display error), so it seems unnecessary.  An alternative could be
```rust
        let dstv = dstv.expect("Could not read DSTV");
```

This would add the note, display the downstream error, and fail the test if the file did not read.

I have no strong feelings one way or another on this change, though.

Reply:
I stand corrected :). It didn't cross my mind that the `unwrap` also displays the downstream error. In that case, checking for 'is_ok()' is redundant indeed.